### PR TITLE
fix for nil max hrs alone

### PR DESCRIPTION
**Pull request by markottaviani**
@TheDudeWithTheThing is there a better control structure to do for this?  Or is this a good choice?

### CODE REVIEW

**Comment by TheDudeWithTheThing** [comment]
Use a view helper to take the logic out of the template 

http://mixandgo.com/blog/the-beginner-s-guide-to-rails-helpers

**Comment by TheDudeWithTheThing** [comment]
Then that becomes 1 span which calls max_hours_alone_css_class(max_hrs_alone) in the class= 

Probably can have a better name than that?

**Comment by markottaviani** [comment]
Added some tests, merging and deploying this so the problem is fixed, going to refactor with helpers in a fresh branch.

File: spec/features/view_adoption_app_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    sign_in(admin)
+
+    visit '/adopters'
+    expect(page).to have_content('Adoption Applications')
+    click_link(adopter_with_null_app.name)
+    expect(page).to have_content(adopter_with_null_app.name)
+
+  end
+end
```
Comment (TheDudeWithTheThing) [inline spec/features/view_adoption_app_spec.rb]: Nooooo!

What editor? Make it always save with a beeline as the last char

Comment (markottaviani) [inline spec/features/view_adoption_app_spec.rb]: using Sublime Text 2, ah ah, history lesson found: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Pull request merged**